extension/data.ts

```diff
@@ -417,7 +623,7 @@ class RcxDict {
 
     while (word.length > 0) {
       const showInf = count !== 0;
-      let trys;
+      let trys: Deinflection[];
```

Comment:
maybe trys -> possibleDeinflections would be good.

Reply:
Totally agree 💯